### PR TITLE
Login Redesign

### LINKED ISSUE #3: Frontend bootstrap templates
Let's discuss about frontend tools. I prefer bootstrap 4 because it's easy to use and look pretty good.

### PR DESCRIPTION
**Pull request by pianokidd03**
### Description
Bootstrap-themed redesign of the login page (registration redesign coming soon). Also fixed the static files bug so that css and image files can be referenced and displayed without a 404 error. Finally the nav was anchored to the bottom of the page.

Fixes #3

### Type of Change:

- Code : 
  - Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by l3str4nge** [comment]
@pianokidd03 Please merge `master` branch with your `fork/master` and resolve conflicts.

**Comment by pianokidd03** [comment]
I hope everything is resolved :)

File: bookomate/templates/base.html [hunk 1]
```diff
@@ -11,6 +11,13 @@
         <link rel="stylesheet" href="{% static 'css/style.css' %}">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <script src="https://kit.fontawesome.com/189832a678.js" crossorigin="anonymous"></script>
+<<<<<<< HEAD
```
Comment (l3str4nge) [inline bookomate/templates/base.html]: Please remove `git diff` indicators.

**Review by l3str4nge — CHANGES_REQUESTED** [review]

File: bookomate/templates/base.html (hunk 1 above)
Comment (pianokidd03) [reply bookomate/templates/base.html]: Sorry, this is a super noob question to ask, but how do I approve your requested changes?

Comment (l3str4nge) [reply bookomate/templates/base.html]: You cannot now because I didn't make changes. Just remove these indicators on your local copy and push changes :)

Comment (pianokidd03) [reply bookomate/templates/base.html]: Ok, the changes have been pushed!

**Review by l3str4nge — APPROVED** [review]
Nice! Thanks for the contribution!

**Pull request merged**